.github/cloud-samples-tools/cmd/main.go

```diff
@@ -29,8 +29,8 @@ import (
 var usage = `usage: tools <command> ...
 
 commands:
-  affected path/to/config.jsonc path/to/diffs.txt
-  run-all path/to/config.jsonc path/to/script.sh
+  affected path/to/config.jsonc diffs.txt paths.txt
+  setups path/to/config.jsonc paths.txt
```

Comment:
nit: `setups` sounds odd as a command in this context. Maybe `set-up`? Or something else?

Reply:
How about `find-setups` or `setup-files`?